packages/firestore/test/unit/specs/write_spec.test.ts

```diff
@@ -1127,4 +1127,21 @@ describeSpec('Writes:', [], () => {
         hasPendingWrites: true
       });
   });
+
+  specTest('Write is sent by newly started client', ['multi-client'], () => {
+    return client(0)
+      .expectPrimaryState(true)
+      .client(1)
+      .expectPrimaryState(false)
+      .userSets('collection/a', { v: 1 })
+      .client(0)
+      .shutdown()
+      .client(2)
+      .expectPrimaryState(true)
+      .writeAcks('collection/a', 1000, { expectUserCallback: false })
```

Comment:
Could we add a `.expectNumOutstandingWrites(1)` here?  It's probably not necessary (`writeAcks()` would hopefully fail if there's no outstanding write), but it'd make the test clearer, I think.